### PR TITLE
chore(vscode): add workspace settings.json

### PR DESCRIPTION
**Pull request by JFrankfurt**
https://code.visualstudio.com/docs/getstarted/settings#_workspace-settings

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "files.autoSave": "afterDelay",
+  "files.autoSaveDelay": 2500,
```
Comment (zzmp) [inline .vscode/settings.json:3]: Please no autosave

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "npm.packageManager": "yarn",
+  "typescript.updateImportsOnFileMove.enabled": "always",
+  "files.exclude": {
+    "**/.git": true,
+    "**/.next": true,
+    "**/.DS_Store": true
+  },
+  "javascript.updateImportsOnFileMove.enabled": "always",
+  "vetur.experimental.templateInterpolationService": true,
+  "workbench.sideBar.location": "right",
```
Comment (zzmp) [inline .vscode/settings.json:13]: Leave user prefs out

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  "editor.inlineSuggest.enabled": true,
+  "editor.codeActionsOnSave": {
+    "source.fixAll": true
+  },
+  "files.eol": "\n",
+  "eslint.enable": true,
+  "eslint.debug": true,
+  "[typescript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  }
```
Comment (zzmp) [inline .vscode/settings.json:27]: Omg this is the config setting I've been looking for for months

**Review by zzmp — CHANGES_REQUESTED** [review]

**Review by zzmp — APPROVED** [review]

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
+  "npm.packageManager": "yarn",
+  "typescript.updateImportsOnFileMove.enabled": "always",
+  "files.exclude": {
+    "**/.git": true,
+    "**/.next": true,
```
Comment (NoahZinsmeister) [inline .vscode/settings.json:6]: dont need this line

**Review by NoahZinsmeister — APPROVED** [review]
lgtm!

**Pull request merged**